E01-Geometry/Benas/src/Shape.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+
+abstract class Shape
```

Comment:
Abstract class with no inherited code is an interface.
Suggesting to change this into interface.